lib/portlayer/exec/container.go

```diff
@@ -412,167 +263,34 @@ func (c *Container) start(ctx context.Context) error {
 	// get existing state and set to starting
 	// if there's a failure we'll revert to existing
 	finalState := c.updateState(StateStarting)
-
 	defer func() { c.updateState(finalState) }()
 
-	// Power on
-	_, err := tasks.WaitForResult(ctx, func(ctx context.Context) (tasks.Task, error) {
-		return c.vm.PowerOn(ctx)
-	})
-	if err != nil {
-		return err
-	}
-
-	// guestinfo key that we want to wait for
-	key := fmt.Sprintf("guestinfo.vice..sessions|%s.started", c.ExecConfig.ID)
-	var detail string
-
-	// Wait some before giving up...
-	ctx, cancel := context.WithTimeout(ctx, propertyCollectorTimeout)
-	defer cancel()
-
-	detail, err = c.vm.WaitForKeyInExtraConfig(ctx, key)
-	if err != nil {
-		return fmt.Errorf("unable to wait for process launch status: %s", err.Error())
-	}
+	err := c.containerBase.start(ctx)
```

Comment:
Good catch. We should have an explicit operation here to set it to known state. Harder with the final error on the stop path.